aws/README.md

```diff
@@ -20,10 +20,20 @@ create create new VPC with public internet access and set also the following env
 export VPC_ID="REPLACE_ME"
 export SUBNET_ID="REPLACE_ME"
 ```
-
+   
 - Create a custom AMI based on Ubuntu 20.04 having kata-agent and other dependencies
-        - **NOTE:** If authentication against container image registries is required copy your JSON format credentials
-        file to [podvm/files/](../podvm/files/)auth.json and add also `USE_SKOPEO=true` before `make image`
+- **Setting up authenticated registry support:** 
+  - If authentication against container image registries is required copy your JSON format credentials file to [podvm/files/etc/](../podvm/files/etc/)auth.json 
```

Comment:
it seems podvm/copy-files.sh doesn't copy this file, isn't it? i guess i missing something